staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -189,6 +196,12 @@ func (o *DebugOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&o.Replace, "replace", o.Replace, i18n.T("When used with '--copy-to', delete the original Pod."))
 	cmd.Flags().StringToString("env", nil, i18n.T("Environment variables to set in the container."))
 	cmd.Flags().StringVar(&o.Image, "image", o.Image, i18n.T("Container image to use for debug container."))
+	cmd.Flags().BoolVar(&o.KeepLabels, "keep-labels", o.KeepLabels, "If true, keep the original pod labels")
```

Comment:
Although there are 6 flags will be introduced (we don't usually prefer), I don't think there is any other alternative way to manage this. Besides, I think, these flags are essential for `kubectl debug` command.